Conch/json-schema/v1.yaml

```diff
@@ -1032,3 +1031,43 @@ definitions:
     type: array
     items:
       $ref: "#/definitions/DeviceService"
+  OrcLifecycle:
+    type: object
+    additionalProperties: false
+    required:
+      - id
+      - name
+      - version
+      - locked
+      - role_id
+      - plan
+      - created
+      - updated
+    properties:
+      id:
+        type: string
+        format: uuid
+      name:
+        type: string
+      created:
+        type: string
+        format: date-time
+      updated:
+        type: string
+        format: date-time
+      version:
+        type: number
+      role_id:
+        type: string
+        format: uuid
+      locked:
+        type: number
+      plan:
+        type: array
+        items:
+          type: string
+          format: uuid
+  OrcLifecycles:
+    type: array
+    items:
+      $ref: "#/definitions/OrcLifecycles"
```

Comment:
Should be singular.

Reply:
I grow increasingly confused as to how schema based tests are passing with typos like this. For instance, the code uses this schema definition in two places in `#all` and the test for `#all`, both of which work just fine.